packages/element/README.md

```diff
@@ -262,6 +262,12 @@ const placeholderLabel = Platform.select( {
 } );
 ```
 
+### PureComponent
+
+_Related_
+
+-   <https://reactjs.org/docs/react-api.html#reactpurecomponent>
```

Comment:
Perhaps we should link to the new docs:

```suggestion
-   <https://react.dev/reference/react/PureComponent>
```

Reply:
> Perhaps we should link to the new docs:

Why not, but let's do it for _all_ references, in its own PR. Searching for `"reactjs.org/docs"` shows me 100 references, in various JSDoc comments and Markdown documents.

Reply:
Sounds good, although I didn't want to burden you with doing it for all existing usages 😅